test/typedarray21.js

```diff
@@ -0,0 +1,22 @@
+// generator: none
```

Comment:
It does - but the resulting code doesn't run in Node (yet):

TypeError: Object #<Int32Array> has no method 'copyWithin'

Or am I missing something? Let me know ;)